src/CryptoFraudDetection/scraper/twitter.py

```diff
@@ -7,3 +7,304 @@
 Authors:
 - 
 """
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+import CryptoFraudDetection.scraper.utils as utils
+from CryptoFraudDetection.utils.exceptions import (
+    DetectedBotException,
+    InvalidParameterException,
+)
+from CryptoFraudDetection.utils.logger import Logger
+
+from typing import List, Tuple, Optional
+import pandas as pd
+import time
+import re
+import json, os
+import csv
+import random
+
+class TwitterScraper:
+    def __init__(self, username: str, password: str, logger: Logger):
+        """
+        Initializes the TwitterScraper class with login credentials and logger.
+
+        Args:
+            username (str): The Twitter username.
+            password (str): The Twitter password.
+            logger (Logger): Logger for logging messages.
+        """
+        self.username = username
+        self.password = password
```

Comment:
make these optional, and throw custom error (you can create one under utils/exceptions.py) if one is trying to scrape without cookie or without user/pass.